specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json

```diff
@@ -7171,6 +7213,41 @@
         }
       }
     },
+    "RegistrationInfoList": {
+      "type": "object",
+      "title": "RegistrationInfoList",
+      "description": "List of RegistrationInfo definitions.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of RegistrationInfo definitions.",
+          "items": {
+            "$ref": "#/definitions/RegistrationInfoMinimal"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Link to the next page of results."
+        }
+      }
+    },
+    "RegistrationInfoMinimal": {
+      "description": "Represents a Minimal set of properties for RegistrationInfo definition.",
+      "type": "object",
+      "properties": {
+        "expirationTime": {
+          "format": "date-time",
+          "type": "string",
+          "description": "Expiration time of registration token.",
+          "x-nullable": true
+        },
+        "token": {
+          "type": "string",
```

Comment:
>string [](http://example.com/codeflow?start=19&length=6)

Add "format": "byte" ?

Reply:
I'm trying to keep this exactly the same as the existing registrationToken definition since this is a security fix going straight to production